tiler-imposm/start.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+mkdir -p /tmp
+
+FILE='http://download.geofabrik.de/south-america/peru-latest.osm.pbf'
```

Comment:
@Rub21 is this importing a pbf file and not from the osm-seed API database? @batpad

Reply:
It is only for test, I will add more code to get the osm-seed replication data and put into the postgis DB.